bgpd/bgp_aspath.c

```diff
@@ -479,7 +479,7 @@ unsigned int aspath_has_as4(struct aspath *aspath)
 }
 
 /* Convert aspath structure to string expression. */
-static void aspath_make_str_count(struct aspath *as)
+static void aspath_make_str_count(struct aspath *as, bool make_json)
```

Comment:
Not that there is any real rule against it, bgpd code doesn't typically use bool type.  From a code consistency perspective, I'm not a fan - from a strict conventions standpoint, we don't have any rule against its use...